io.github.papoteur-mga.Elograf.yaml

```diff
@@ -0,0 +1,60 @@
+id: io.github.papoteur-mga.Elograf
```

Comment:
the app-id is still invalid: see https://docs.flatpak.org/en/latest/conventions.html#application-ids

(`-` as not allowed where you have one)

Also it must be changed in the metainfo.xml